sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -296,8 +296,23 @@ private[ui] class ExecutionPagedTable(
 
     require(sortableColumnHeaders.contains(sortColumn), s"Unknown column: $sortColumn")
 
+    def getHeaderTooltips(header: String): String = header match {
+      case "ID" => ToolTips.SQL_TAB_ID
+      case "Description" => ToolTips.SQL_TAB_DESCRIPTION
+      case "Submitted" => ToolTips.SQL_TAB_SUBMITTED
+      case "Duration" => ToolTips.SQL_TAB_DURATION
+      case "Running Job IDs" => ToolTips.SQL_TAB_RUNNING_JOB_IDS
+      case "Succeeded Job IDs" => ToolTips.SQL_TAB_SUCCEEDED_JOB_IDS
+      case "Failed Job IDs" => ToolTips.SQL_TAB_FAILED_JOB_IDS
+      case "Job IDs" => header
+      case unknownColumn => throw new IllegalArgumentException(s"Unknown column: $unknownColumn")
+    }
```

Comment:
Can we have a simple map here instead of `getHeaderTooltips` and `ToolTips.scala`? Currently, it seems too much for tooltips.

Reply:
ok i will change it to map it will be good.

Reply:
Thanks!